integration-distro-test/pom.xml

```diff
@@ -0,0 +1,177 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright 2015 Red Hat, Inc. and/or its affiliates
+    and other contributors as indicated by the @author tags.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.hawkular</groupId>
+    <artifactId>hawkular-parent</artifactId>
+    <version>5</version>
+  </parent>
+
+  <artifactId>hawkular-integration-test</artifactId>
+  <version>1.0.0.Alpha1-SNAPSHOT</version>
+  <packaging>pom</packaging>
+  <name>Hawkular Integration Distribution</name>
+
+  <properties>
+    <checkstyle.suppressions.file>${basedir}/src/main/scripts/checkstyle-suppressions.xml</checkstyle.suppressions.file>
+
+    <!-- The versions of the components that go into our integration distribution -->
+    <version.org.hawkular.nest>1.0.0.Alpha1-SNAPSHOT</version.org.hawkular.nest>
+    <version.org.hawkular.inventory>1.0-SNAPSHOT</version.org.hawkular.inventory>
+    <version.org.hawkular.alerts>1.0.0.Alpha1-SNAPSHOT</version.org.hawkular.alerts>
+    <version.org.rhq.metrics>0.2.8-SNAPSHOT</version.org.rhq.metrics>
```

Comment:
We need 0.2.8-SNAPSHOT. 0.2.7 had a bug in metrics that I fixed and Stefan merged into metrics yesterday.